tests/atomistics/structure/factories/test_atomsk.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+import shutil
+
+from pyiron_atomistics._tests import TestWithProject
+from pyiron_atomistics.atomistics.structure.factories.atomsk import AtomskFactory, AtomskError
+
+if shutil.which("atomsk") is not None:
```

Comment:
I'm a little uncomfortable hard-coding the executable name (e.g. what if someone has multiple atomsk versions installed and renames them accordingly?) but I don't think it needs to stop the merge.

Reply:
Agree, ideally this would somehow reuse the `Executable` class that we have already for jobs, so you'd be able to configure your particular atomsk, but I'm drawing a blank how to actually do it right now.

Reply:
Fair!